doc/api/http.md

```diff
@@ -2081,6 +2092,7 @@ not abort the request or do anything besides add a `'timeout'` event.
 [`http.Server`]: #http_class_http_server
 [`http.globalAgent`]: #http_http_globalagent
 [`http.request()`]: #http_http_request_options_callback
+[`http.get()`]: #http_http_get_options_callback
```

Comment:
Nit: bottom references also sorted in ASCII order, so this one seems to go before `[`http.globalAgent`]: #http_http_globalagent`.